packages/cozy-procedures/src/templates/creditApplicationTemplate.js

```diff
@@ -144,7 +144,9 @@ const documents = {
     rules: {
       metadata: {
         classification: 'tax_notice',
-        subject: 'income'
+        subjects: {
+          $in: ['income']
```

Comment:
I would have used `$elemMatch` here, but cool if `$in` works as well.

Reply:
I can confirm, it works has expected :) 
<img width="739" alt="Capture d’écran 2019-07-02 à 10 37 43" src="https://user-images.githubusercontent.com/1107936/60497848-e8303d80-9cb5-11e9-899e-1b85f5aca847.png">